app/javascript/components/layout/Footer.jsx

```diff
@@ -1,31 +1,15 @@
 import React from 'react';
 import Email from 'components/icons/Email';
-import LinkedIn from 'components/icons/LinkedIn';
+import Github from 'components/icons/Github';
 import Twitter from 'components/icons/Twitter';
 
 import 'stylesheets/footer.scss';
 
-/* TODO: re-enable this rule after all links are in place */
-/* eslint-disable jsx-a11y/anchor-is-valid */
-
 const Footer = () => (
     <footer className="footer">
         <div className="footer-left">
             <div className="footer-col">
-                <p className="footer-col-item footer-col-title">WNB.rb Meetup</p>
```

Comment:
Here is how I've reorganized the footer:

* Get Involved
  * Join WNB.rb
  * Sponsor Us
  * (I would put a donate link here for community members to give us money)
* Community Guidelines (not sure if I love this header)
  * Code of Conduct
  * Slack Guidelines
* Resources (not sure I love this header either)
  *  Job Board
  * (Would put "Our next meetup" and "Past meetups" links here once they're ready)

I am SO open to feedback and thoughts here.